src/pipelines/rust.rs

```diff
@@ -337,8 +337,13 @@ impl RustApp {
             args.push("--weak-refs");
         }
 
+        if !self.typescript {
+            args.push("--no-typescript");
+        }
+
         // Invoke wasm-bindgen.
         tracing::info!("calling wasm-bindgen for {}", self.name);
+        tracing::info!("wasm-bindgen args: {:#?}", &args);
```

Comment:
Could probably be lowered to `debug`, but that shouldn't be a blocker for merging.
Will change this later myself.